docs/specs/metadata.yml

```diff
@@ -28,8 +28,113 @@ inputs:
     # Title
 outputs:
   docs/a.json: |
-    { "ms.custom-metadata": null, "author":"xxx", "uid": null, "breadcrumb_path": "/breadcrumb_path", "title": "Title"}
+    { 
+      "ms.custom-metadata": null, 
+      "author":"xxx", 
+      "uid": null, 
+      "breadcrumb_path": "/breadcrumb_path", 
+      "title": "Title",    
+    }
+---
+# output _xrefmap per file to structure json
+repos:
+  https://docs.com/test#test:
```

Comment:
If the test does not test repo related features, use `inputs` to speed up test, because `repos` would create an actually git repository on disk and is expensive.

Reply:
the `repos` used here is for providing the **branch** name which will be used to construct absolute URL in xref href.